nemo_text_processing/inverse_text_normalization/fr/data/electronic/symbols.tsv

```diff
@@ -0,0 +1,12 @@
+chez	@
+at	@
+à	@
+arobase	@
+point	.
+barre oblique	/
+tiret	-
+tiret bas	_
+souligné	_
+sous-tiret	_
+blanc souligné	_
+underscore	_
```

Comment:
underscore?

Reply:
Internet related categories appear to adopt some Anglicisms into vocabulary. Helps improve coverage.